### PR TITLE
PGO: Update the Helix payload to rely on the unpackaged distribution

### PR DESCRIPTION
**Pull request by DHowett**
The unpackaged distribution was made for this exact use, so let's *do it*!

### CODE REVIEW

**Comment by DHowett** [comment]
The PGO build is marked as failed because I cancelled it after it produced pgc files properly. :smile:

**Comment by DHowett** [comment]
This leaves the localtests broken in Helix, which I'll get to afterwards.

**Review by zadjii-msft — APPROVED** [review]

File: build/Helix/PrepareHelixPayload.ps1 [hunk 1]
```diff
@@ -55,14 +55,7 @@ Copy-Item "build\helix\runtests.cmd" $payloadDir
...
-# and refuses to unzip it because of its file extension while on a desktop, it just
-# does the job without complaining.
-
-# Extract the APPX package
-Expand-Archive -LiteralPath $payloadDir\CascadiaPackage.zip -DestinationPath $payloadDir\appx
-Expand-Archive -LiteralPath $payloadDir\VCLibs.zip -DestinationPath $payloadDir\appx -Force
+# Extract the unpackaged distribution of Windows Terminal to the payload directory,
+# where it will create a subdirectory named terminal-0.0.1.0
+# This is referenced in TerminalApp.cs later as part of the test harness.
+& tar -x -v -f "$repoDirectory\Artifacts\$ArtifactName\unpackaged\WindowsTerminalDev_0.0.1.0_x64.zip" -C "$payloadDir"
```
Comment (lhecker) [inline build/Helix/PrepareHelixPayload.ps1:61]: I think --full options are better for CI scripts for clarity.

**Review by lhecker — APPROVED** [review]

File: build/Helix/PrepareHelixPayload.ps1 (hunk 1 above)
Comment (DHowett) [reply build/Helix/PrepareHelixPayload.ps1:61]: ah so, _the canonical options for tar are the short ones_. windows tar doesn't support the GNU-style modern `--long-options` :)

**Pull request merged**